store/firstlast.go

```diff
@@ -38,3 +38,13 @@ func Last(st KVStore, start, end []byte) (kv cmn.KVPair, ok bool) {
 
 	return cmn.KVPair{Key: iter.Key(), Value: iter.Value()}, true
 }
+
+//----------------------------------------
+func cp(bz []byte) (ret []byte) {
```

Comment:
At this point we have several instances of this exact function in the codebase, can we de-duplicate them and put it in `types/` ?